.github/workflows/build.yml

```diff
@@ -17,14 +17,48 @@ jobs:
       CCACHE_MAXSIZE: 0
       CCACHE_NOCOMPRESS: 1
       MAKEFLAGS: -j2
+      DEBIAN_FRONTEND: noninteractive
     steps:
       - name: Configure git
-        run: git config --global --add safe.directory "$GITHUB_WORKSPACE"
+        run: |
+          apt update && apt install sudo
+          sudo apt-get install --yes git
+          git config --global --add safe.directory "$GITHUB_WORKSPACE"
       - name: Check out
         uses: actions/checkout@v3
         with:
           fetch-depth: 0
           submodules: recursive
+      - name: Prepare to install system packages
+        run: |
+          echo "APT_DEPENDENCIES=$(xargs < .github/workflows/deps-ubuntu-20.04-gcc.txt)" >> "$GITHUB_ENV"
+          .github/workflows/setup-wine.sh
+          .github/workflows/setup-cmake.sh
```

Comment:
I don't understand getting cmake from the PPA either. Is apt version outdated?

Reply:
Yes, Ubuntu 20.04 is 4.5 years old, so everything is out of date